grammars/php.cson

```diff
@@ -2041,6 +2041,15 @@
         'match': ','
         'name': 'punctuation.separator.delimiter.php'
       }
+      {
```

Comment:
Is there any particular reason for the includes? Could those simply be just inlined?

Reply:
Personal preference. Never know when they might be needed to compose some larger expression in the future. Ran into that problem a couple times with inlined matches in `language-css`.